manage-people/routes/PeopleRoutes.js

```diff
@@ -2,20 +2,20 @@
 
 import express from 'express';
 import generateSafeId from 'generate-safe-id';
-import bodyParser from 'body-parser'; 
+import bodyParser from 'body-parser';
 import Person from './../domain/Person'
 
 let router = express.Router();
 
 const peopleRoutes = (app) => {
 
-    app.use(bodyParser.json())
+    app.use(bodyParser.json());
 
     router
         .get('/', (req, res) => {
             Person.scan().exec(function (err, data) {
                 if(err) {
-                    return res.status(500).send({message: 'Doh...'});
+                    return res.status(500).send({message: err});
```

Comment:
I think providing technical details to end users is a big lack of security, that's why I was sending a Doh... instead. Can we keep doing that or do we have a reason to send the error.

Reply:
Okay, so I'll change it back to Doh again.

Reply:
Actually the end users of TinyUI are us; all super technical. A nice solution would be to have a systematic logging solution and log tons of error details on any error and display a URL that opens a nice log browser. That of course would be terrible for a real end user UI but it would be great for Tiny UI. As for "security"; to logging system should be protected but the URL that appears in the UI does not have to reveal anything more than a guid.